docs_nnx/guides/pytree.ipynb

```diff
@@ -395,8 +395,8 @@
    "id": "8055b72c",
    "metadata": {},
    "source": [
-    "### Class Annotations\n",
-    "Annotations can also be added at the type level via `nnx.Static` and `nnx.Data`. This is useful for creating `dataclasses` but the mechanism also works for regular classes."
+    "### Dataclasses\n",
+    "To create `nnx.Pytree` dataclasses, `nnx.static` and `nnx.data` can be used as `field` specifiers for `nnx.dataclass`."
```

Comment:
What do you think of starting this paragraph with something like?:
  "The `nnx.dataclass` annotation can be used to create a subclass of `nnx.Pytree` that is also a dataclass."
or:
  "To create a dataclass that is also an `nnx.Pytree`, use the `nnx.dataclass` annotation."
(And then mentioning `nnx.static` and `nnx.data` in the next sentence.)